geode-web/src/test/java/org/apache/geode/management/internal/cli/commands/AlterRuntimeCommandDUnitTest.java

```diff
@@ -111,13 +112,622 @@ public void testAlterRuntimeConfig(final boolean connectOverHttp) throws Excepti
+    assertThat(gfsh.getGfshOutput()).contains("Could not set \"log-file-size-limit\" to \"-1\"");
+
+    for (MemberVM server : new MemberVM[] {server1, server2}) {
+      server.invoke(() -> {
+        InternalCache cache = LocatorServerStartupRule.serverStarter.getCache();
+        DistributionConfig config = cache.getInternalDistributedSystem().getConfig();
+        assertThat(config.getLogFileSizeLimit()).isEqualTo(0);
+        assertThat(config.getArchiveDiskSpaceLimit()).isEqualTo(0);
+        assertThat(config.getStatisticSampleRate()).isEqualTo(1000);
+        assertThat(config.getStatisticArchiveFile().getName()).isEqualTo("");
+        assertThat(config.getStatisticSamplingEnabled()).isTrue();
+        assertThat(config.getLogDiskSpaceLimit()).isEqualTo(0);
+      });
+    }
+  }
+
+  @Test
+  @Parameters({"true", "false"})
+  public void alterLogFileSizeLimitTooBig_errorCanNotSet(final boolean connectOverHttp)
+      throws Exception {
+    IgnoredException.addIgnoredException(
+        "java.lang.IllegalArgumentException: Could not set \"log-file-size-limit\"");
+
+    Properties props = new Properties();
+    props.setProperty(LOG_LEVEL, "error");
+    MemberVM locator = startupRule.startLocatorVM(0, props);
+    MemberVM server1 = startupRule.startServerVM(1, props, locator.getPort());
+    props.setProperty(GROUPS, "G1");
+    MemberVM server2 = startupRule.startServerVM(2, props, locator.getPort());
+
+    if (connectOverHttp) {
+      gfsh.connectAndVerify(locator.getHttpPort(), GfshShellConnectionRule.PortType.http);
+    } else {
+      gfsh.connectAndVerify(locator.getJmxPort(), GfshShellConnectionRule.PortType.jmxManger);
+    }
+
+    CommandStringBuilder csb = new CommandStringBuilder(CliStrings.ALTER_RUNTIME_CONFIG);
+    csb.addOption(CliStrings.ALTER_RUNTIME_CONFIG__LOG__FILE__SIZE__LIMIT, "1000001");
+    CommandStringBuilder csbGroup = new CommandStringBuilder(csb.toString());
+    csbGroup.addOption(GROUPS, "G1");
+
+    CommandResult result = gfsh.executeCommand(csb.toString());
+    assertThat(result.getStatus()).isEqualTo(Result.Status.ERROR);
+    assertThat(gfsh.getGfshOutput())
+        .contains("Could not set \"log-file-size-limit\" to \"1,000,001\"");
+
+    csb.addOption(CliStrings.MEMBER, server2.getName());
+    result = gfsh.executeCommand(csb.toString());
+    assertThat(result.getStatus()).isEqualTo(Result.Status.ERROR);
+    assertThat(gfsh.getGfshOutput())
+        .contains("Could not set \"log-file-size-limit\" to \"1,000,001\"");
+
+    result = gfsh.executeCommand(csbGroup.toString());
+    assertThat(result.getStatus()).isEqualTo(Result.Status.ERROR);
+    assertThat(gfsh.getGfshOutput())
+        .contains("Could not set \"log-file-size-limit\" to \"1,000,001\"");
+
+    for (MemberVM server : new MemberVM[] {server1, server2}) {
+      server.invoke(() -> {
+        InternalCache cache = LocatorServerStartupRule.serverStarter.getCache();
```

Comment:
It looks like this exact block gets invoked a few times throughout the test (lines #357, #402, #708).  It might make things simpler to extract it into a method